hazelcast/src/main/java/com/hazelcast/quorum/impl/QuorumServiceImpl.java

```diff
@@ -224,12 +226,18 @@ public void ensureQuorumPresent(Operation op) {
         quorum.ensureQuorumPresent(op);
     }
 
-    public void ensureQuorumPresent(String quorumName, QuorumType requiredQuorumPermissionType) {
+    @Override
+    public void ensureQuorumPresent(@Nullable String quorumName,
+                                    @Nonnull QuorumType requiredQuorumPermissionType) {
+        checkNotNull(requiredQuorumPermissionType, "requiredQuorumPermissionType cannot be null!");
         if (isInactive() || quorumName == null) {
             return;
         }
 
         QuorumImpl definedQuorum = quorums.get(quorumName);
+        if (definedQuorum == null) {
```

Comment:
This was a fix where there was a NPE if there was no configured quorum with the provided `quorumName`. I assumed the same behaviour as when there is no configured qourum (`isInactive() == true` above).

Reply:
Just a note for posterity: in `SplitBrainProtectionServiceImpl` now.